pkg/ctxlog/events.go

```diff
@@ -112,7 +112,10 @@ func (ev Event) debugJSON(ctx context.Context, objectInfo interface{}) {
 
 	// treat JSON data as a string map and extract message
 	var result map[string]string
-	json.Unmarshal([]byte(jsonData), &result)
+	if err := json.Unmarshal([]byte(jsonData), &result); err != nil {
+		return
```

Comment:
Maybe it would be helpful to print the error to debug if it fails?

Reply:
Maybe it was silent for a reason, but then a few lines done it does print a debug msg. So why not here, too.